### ISSUE TITLE
SSL_ERROR_RX_RECORD_TOO_LONG

### ISSUE THREAD

**Issue by Nv7-GitHub** [FiloSottile/mkcert#361]
I made a certificate using `mkcert -install`. I then went to `https://localhost:8080`, using the test webserver I was running. It worked with `http://localhost:8080`, but not with `https://localhost:8080`. It gave me the error `SSL_ERROR_RX_RECORD_TOO_LONG`. I have attached a screenshot.
It seems like the SSL certificate was successfully created though, I have also attached a screenshot.
<img width="618" alt="Screen Shot 2021-04-05 at 9 43 31 PM" src="https://user-images.githubusercontent.com/50369270/113659980-0891e680-9658-11eb-9eb2-a654a32ffec9.png">

<img width="897" alt="Screen Shot 2021-04-05 at 9 42 39 PM" src="https://user-images.githubusercontent.com/50369270/113659898-df715600-9657-11eb-84b5-5211b4f85f5b.png">

**Comment by nazmulpcc** [comment]
Having the same issue. I am generating the certificate from a dockerized Ubuntu 20.4 and my host system is Manjaro Linux. I am using mkcert 1.1.2.

**Comment by sgalcheung** [comment]
> localhost:5001 uses an invalid security certificate.
>  
> The certificate is not trusted because it is self-signed.
>  
> Error code: MOZILLA_PKIX_ERROR_SELF_SIGNED_CERT

**Comment by kenricashe** [comment]
Same here using 1.4.3 in Ubuntu 20.10 with Apache 2.4.46. I followed the instructions from [Chandan Kumar](https://geekflare.com/local-dev-environment-ssl/) and had no errors until trying to actually load in Firefox.

**Comment by kenricashe** [comment]
Found my solution!

Just had to [configure support for TLS 1.2 and 1.3 in Apache2](https://tecadmin.net/enable-tls-version-apache/).

**Comment by ivanjaros** [comment]
yep, same problem on windows 10 and ff 75.0.2 . i have built from source(go). last commit was in february.

**Comment by dmitryuk** [comment]
same issue, but finally found I missed to add ssl flag within nginx config

<img width="359" height="96" alt="Image" src="https://github.com/user-attachments/assets/8d0f86b9-22a2-4e6f-befb-3990cd5af567" />
